pygfx/objects/_base.py

```diff
@@ -400,6 +403,26 @@ def look_at(self, target: Vector3):
             self._q.set_from_rotation_matrix(self._m)
             self.rotation.premultiply(self._q.inverse())
 
+    def get_world_position(self):
+        self.update_matrix_world(update_parents=True, update_children=False)
+        self._v.set_from_matrix_position(self._matrix_world)
+        return self._v.clone()
+
+    def get_world_bounding_box(self, include_children=True):
+        self.update_matrix_world(update_parents=True, update_children=False)
+        aabb = self._geometry.bounding_box()
+        aabb_world = transform_aabb(aabb, self._matrix_world.to_ndarray())
+        if self._children and include_children:
+            aabbs = np.stack(
+                [c.get_world_bounding_box() for c in self._children] + [aabb_world]
+            )
+            aabb_world = np.array([aabbs[:, 0].min(axis=0), aabbs[:, 1].max(axis=1)])
+        return aabb_world
+
+    def get_world_bounding_sphere(self, include_children=True):
+        aabb = self.get_world_bounding_box(include_children=include_children)
+        return aabb_to_sphere(aabb)
```

Comment:
Wouldn't it be better to actually calculate with spheres here? How much harder would that be?

Reply:
It's non trivial to the point I would like to settle for this now and maybe revisit it later :)

Reply:
Agreed :)